__tests__/app.test.js

```diff
@@ -19,6 +19,27 @@ describe("GET /api/topics", () => {
                 })
             })
     })
+    test("200 - returns documentation on available endpoints", () => {
+        return request(app)
+            .get("/api")
+            .expect(200)
+            .then(({ body }) => {
```

Comment:
You can just test that the endpoints equals the endpoints file. This may seem redundant but what we are testing here is that the file contents get sent to the user. The actual contents will be looked over by eye.